integration/localnet/bootstrap.go

```diff
@@ -154,23 +153,7 @@ func main() {
 
 	fmt.Println("Node bootstrapping data generated...")
 
-	// gather access node IDS for LN/SN nodes
-	accessNodeIDS := make([]string, 0)
-	for i, c := range containers {
-		fmt.Printf("%d: %s", i+1, c.Identity().String())
-		if c.Unstaked {
```

Comment:
If you're removing this, you might as well remove the boolean flag `Unstaked` in `integration/testnet/container.go`. It's obsolete anyway. /cc @vishalchangrani for double-check.

Reply:
https://github.com/onflow/flow-go/pull/1426/commits/9abd914a2ab87872409d7993616f732b480b1a8c